integration/scripts/test_logs.py

```diff
@@ -11,23 +12,10 @@ def test_errors_logs(kube_client, helm_config):
     """_summary_
 
     This test does the following:
-    - Check that all pods in the cluster's namespace have no errors/warnings in their logs
-    - Ignore expected errors
+    - Check that all pods in the cluster's namespace have no 'level': 'error' in their json logs
     - Check that all pods have the expected logs
     """
 
-
-    ignore_error_patterns = [
```

Comment:
were all these old messages in the warning level? or they were error level and they'd actually fail a test now?

Reply:
These we old ignore patterns we had to manually exclude these warnings (mainly level:warn)

Reply:
if any of them were level:error then they would now start failing the integration tests since we fail any level:error now right? is that what you want?

Reply:
Correct. These would now fail 
The patterns added were mainly warnings I believe (which had the word "Error" in it)
Instead of checking for presence of "error" and failing that, we just check for `level'

I can update accordingly if we see more failures but the end goal is that `level: error` should be a red flag for us to "do something" vs `level: warn` which we were previously failing on but manually ignoring